lang/en/package.html

```diff
@@ -17,66 +17,61 @@ <h2 class="details-main--title d-inline-block m-2">…</h2>
             <!-- react-text: 8 -->
             <!-- /react-text -->
           </a><span class="ais-Hit--popular null" title="0 downloads in the last 30 days">0</span>
-          <!-- react-empty: 10 --><span class="ais-Hit--version">0.0.0</span></div>
+          <!-- react-empty: 10 --><span class="ais-Hit--version"></span></div>
```

Comment:
Is this file generated through a script? I'm guessing it's the result of rendering the React component to a string (given the presence of ` <!-- react-empty: 10 -->` and such) but it'd be good to have an easy way to keep it up-to-date.